src/App.css

```diff
@@ -29,16 +34,77 @@
 .city{
   font-size: 30px;
   font-weight: 200;
+  font-family: 'Montserrat', sans-serif;
 }
 
 .temp{
-  font-size: 90px;
   padding: 10px;
   border: 1px solid lightgray;
   border-radius: 12px;
+  font-family: 'Montserrat', sans-serif;
+  display: flex;
+  flex-direction: column;
+}
+
+.unitContainer{
+  font-size: 90px;
+  font-family: 'Montserrat', sans-serif;
+
 }
 
 .weather{
   font-size: 30px;
   font-weight: 200;
+  font-family: 'Montserrat', sans-serif;
+}
+
+.welcomeContainer{
+  margin-top: 30px;
+  font-size: 20px;
+  font-family: 'Montserrat', sans-serif;
+}
+
+.errorTemp{
+  padding: 10px;
+  border: 1px solid lightgray;
+  border-radius: 12px;
+  font-family: 'Montserrat', sans-serif;
+  display: flex;
+  flex-direction: column;
+  margin-top: 40px;
+  font-size: 40px;
+  font-weight: bold;
+}
+
+@media only screen and (max-width: 1100px) {
+  body{
+    background-image: url(../src/images/background3.jpg) no-repeat center fixed;
+    background-size: cover;
+  }
+
+  .temp{
+    padding: 10px;
+    border: 1px solid lightgray;
+    border-radius: 12px;
+    font-family: 'Montserrat', sans-serif;
+    display: flex;
+    flex-direction: column;
+    /* background-color: black; */
+  }
+
+  .weather{
+    font-size: 30px;
+    font-weight: 200;
+    font-family: 'Montserrat', sans-serif;
```

Comment:
I see that you're repeating the `font-family` property in almost every declaration you have in there. 

You could add it to your `body` declaration and then not have to repeat it so many times.

You could have something like this:
```
body {
font-family: 'Montserrat', sans-serif;
}
```

Then if you need to overwrite this font with another font then you're going to be able to depend on the CSS cascading to overwrite. (We can discuss this over the weekend).